src/Kafka/Conduit/Sink.hs

```diff
@@ -86,3 +88,14 @@ kafkaSink props =
         Just msg -> do
           res <- produceMessage prod msg
           runHandler $ maybe (Right prod) Left res
+
+commitOffsetsSink :: MonadIO m => KafkaConsumer -> Sink i m ()
+commitOffsetsSink = awaitForever . const . commitAllOffsets OffsetCommit
+
+flushProducerSink :: MonadIO m => KafkaProducer -> Sink i m ()
```

Comment:
It probably can't, but I don't think it should be doing it as a part of a library ;)